lms/djangoapps/teams/models.py

```diff
@@ -319,25 +319,20 @@ def get_memberships(cls, username=None, course_ids=None, team_ids=None):
         return queryset
 
     @classmethod
-    def user_in_team_for_course(cls, user, course_id, topic_id=None):
+    def user_in_team_for_teamset(cls, user, course_id, topic_id=None):
         """
-        Checks user membership in two ways:
-        if topic_id is None, checks to see if a user is assigned to any team in the course
-        if topic_id (teamset) is provided, checks to see if a user is assigned to a specific team in the course.
+        Using the provided topic_id (teamset), checks to see if a user is assigned to a specific team in the course.
```

Comment:
Copy pasta - should be ... to a specific team in the **teamset**.

Reply:
Not sure why it says "specific" team. It's not checking a specific team, it's checking for any team.

Reply:
copy pasta?

Reply:
Actually, less copy pasta, and more me wrapping my brain on how to properly express the behavior.  I've updated the docstring per comments.